### DISCUSSION TITLE
Short URLs list can't turn pages, filter and search

### DISCUSSION THREAD

**Discussion by sxjeru** [shlinkio/discussions#31] [Help wanted] [answered]
#### Shlink web client version

* Version: [3.7.3](https://github.com/shlinkio/shlink-web-client/releases/v3.7.3)
* How do you use shlink-web-client: app.shlink.io

#### Summary

All in the title. I can't do the following on page "List short URLs":
- Go to Page 2
- Sort by header
- Search by text or tag
- Filter by time

Considering that there may be performance concerns. I also try to add Redis to the shlink in Docker. 
But according to the document, I don't know how to fill in the password of Redis.
I have tried to fill `password@server:port` in the `REDIS_SERVERS` environment variable, it failed.

Thanks for any support!

**Comment by acelaya** [comment]
Can you share a recording to see what you mean? What browser are you using? Does the browser console show any error?

**Comment by sxjeru** [comment]
I tried PC chrome (106.0.5249.91), Edge and Android Chrome, and they all had the problem.

The console will output after each refresh: 
`GET https://[domain]/rest/v2/mercure-info 501`

The following sentence appears many times when staying on the page:
`Unchecked runtime.lastError: A listener indicated an asynchronous response by returning true, but the message channel closed before a response was received`

When I turn page or search, the console does not give any errors.

**Comment by acelaya** [comment, marked as answer]
This looks like something between shlink-web-client and your Shlink instance is stripping out the query params, making it always load first page with no filtering.

Do you have a reverse proxy in front of Shlink? Do these request work when calling the API through a different client (postman, https://api-spec.shlink.io, etc)? Do they work if you bypass the proxy?

**Reply by sxjeru** [reply]
It suddenly occurred to me that I added CDN to the server and disabled parameter following. When I enabled it, everything returned to normal.

I'm sorry to bother you, thank you for help!

**Reply by sxjeru** [reply]
In addition, can you help me delete the screen recording in the issue for privacy. I don't have permission to edit.
